Library/Homebrew/cli/parser.rb

```diff
@@ -459,11 +479,7 @@ def hide_from_man_page!
 
       private
 
-      SYMBOL_TO_USAGE_MAPPING = {
```

Comment:
this isn't actually private, i hoisted it and added a `private_constant` declaration